packages/ncids-css/scss/components/nci-header/internal/_navigation.scss

```diff
@@ -0,0 +1,262 @@
+$nav-link-color: "gray-60";
+$nav-link-arrow-icon-size: 2;
+
+// Replicated from USWDS and changed to use the :before selector
+// as the place-icon mixin uses the :after selector - having the underline
+// and icon in the primary nav button.
+@mixin add-before(
```

Comment:
Please do not place mixins in the component styling. I know USWDS does that but this can cause issues down the road.